scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -527,7 +527,8 @@ class Router(formatOps: FormatOps) {
       // Opening ( with no leading space.
       case FormatToken(
             RightParenOrBracket() | T.KwSuper() | T.KwThis() | T.Ident(_) |
-            T.RightBrace() | T.Underscore(),
+            T.RightBrace() | T.Underscore() | T.MacroQuotedIdent(_) |
+            T.MacroSplicedIdent(_),
```

Comment:
so this was rolled back, right?

Reply:
That's right. We remove the additional Idents, because `$a` is actually a valid name.